thirdparty/patches/spdlog-rotation-file-format.patch

```diff
@@ -0,0 +1,30 @@
+commit b19217a482b28c008793940ca154a2f99c5b3fd7
+Author: dentiny <dentinyhao@gmail.com>
+Date:   Thu Dec 19 17:17:56 2024 +0000
+
+    patch spdlog rotation filename
+
+    Signed-off-by: dentiny <dentinyhao@gmail.com>
+
+diff --git a/include/spdlog/sinks/rotating_file_sink-inl.h b/include/spdlog/sinks/rotating_file_sink-inl.h
+index cf8b9d5c..4504b866 100644
+--- a/include/spdlog/sinks/rotating_file_sink-inl.h
++++ b/include/spdlog/sinks/rotating_file_sink-inl.h
+@@ -50,7 +50,7 @@ SPDLOG_INLINE rotating_file_sink<Mutex>::rotating_file_sink(
+ }
+
+ // calc filename according to index and file extension if exists.
+-// e.g. calc_filename("logs/mylog.txt, 3) => "logs/mylog.3.txt".
++// e.g. calc_filename("logs/mylog.txt, 3) => "logs/mylog-3.txt".
```

Comment:
If we do the patch, let's just do `logs/mylog.txt.1`?

Reply:
Sure, updated.